src/dialog/dialog-content.component.html

```diff
@@ -8,11 +8,11 @@
         <ng-content *ngIf="!message"></ng-content>
 
         <div *ngIf="leftButtonLabel || rightButtonLabel" class="igx-dialog__window-actions">
-            <button *ngIf="leftButtonLabel" igxButton="{{ leftButtonType }}" igxButtonColor="{{ leftButtonColor }}" igxButtonBackground="{{ leftButtonBackgroundColor }}"
+            <button *ngIf="leftButtonLabel" type="button" igxButton="{{ leftButtonType }}" igxButtonColor="{{ leftButtonColor }}" igxButtonBackground="{{ leftButtonBackgroundColor }}"
```

Comment:
Shouldn't it be an `<input type="button" />`?

Reply:
The Ripple effect doesn't work on Input, but despite of this after I spoke with @simeonoff , we think it is good to expose the type as 'Input' of the dialog and set it to button by default.